### PR TITLE
rewords usage of decentralized extensions

### LINKED ISSUE #502: `Decentralized extension` is not discussed
In Sec. 4.1 Extensibility,

> 2. Representations MAY deﬁne other extensibility mechanisms including methods for decentralized extensions. Such extension mechanisms MUST support lossless conversion into any other conformant representation.

There is no discussion on what "decentralized extensions" is anywhere in the doc.

### PR DESCRIPTION
**Pull request by kdenhartog**
closes #502

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/514.html" title="Last updated on Jan 3, 2021, 4:36 PM UTC (e636738)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/514/6434f17...e636738.html" title="Last updated on Jan 3, 2021, 4:36 PM UTC (e636738)">Diff</a>

### CODE REVIEW

**Review by peacekeeper — APPROVED** [review]
+1 to this, as proposed in my comment https://github.com/w3c/did-core/issues/502#issuecomment-748699266

File: index.html [hunk 1]
```diff
@@ -1270,11 +1270,11 @@ <h2>Extensibility</h2>
 together.
         </li>
       <li>
-Representations MAY define other extensibility mechanisms including methods for
-decentralized extensions. Such extension mechanisms SHOULD support lossless
-conversion into any other conformant representation. Extension mechanisms for a
-representation SHOULD define a mapping of all properties and representation
-syntax into the <a href="#data-model">data model</a> and its type system.
+Representations MAY define other extensibility mechanisms ncluding ones that do
```
Comment (brentzundel) [inline index.html:1273]:
```suggestion
Representations MAY define other extensibility mechanisms, including ones that do
```

File: index.html [hunk 2]
```diff
@@ -1270,11 +1270,11 @@ <h2>Extensibility</h2>
...
+not require the use of the DID Core Registries. Such extension mechanisms SHOULD
+support lossless conversion into any other conformant representation. Extension
+mechanisms for a representation SHOULD define a mapping of all properties and
+representation syntax into the abstract data model and its type system.
```
Comment (brentzundel) [inline index.html:1277]: should `data model` still be a link?

Comment (kdenhartog) [reply index.html:1277]: Yup, I'll go back in and fix this. Thanks!

Comment (kdenhartog) [reply index.html]: Addressed, thanks for the review.

File: index.html [hunk 3]
```diff
@@ -1270,11 +1270,12 @@ <h2>Extensibility</h2>
 together.
         </li>
       <li>
-Representations MAY define other extensibility mechanisms including methods for
-decentralized extensions. Such extension mechanisms SHOULD support lossless
-conversion into any other conformant representation. Extension mechanisms for a
-representation SHOULD define a mapping of all properties and representation
-syntax into the <a href="#data-model">data model</a> and its type system.
+Representations MAY define other extensibility mechanisms, including ones that do
+not require the use of the DID Core Registries. Such extension mechanisms SHOULD
```
Comment (msporny) [inline index.html:1274]:
```suggestion
not require the use of the DID Specification Registries. Such extension mechanisms SHOULD
```

**Review by msporny — APPROVED** [review]
Minor nits, otherwise LGTM.

**Review by rhiaro — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**